js/dashboard.js

```diff
@@ -45,157 +44,26 @@ function resetResults(){
-  document.getElementById("update-box").hidden=false;
-  
-  //set values of update input boxes to values from database
-  document.getElementById("update-First-name").value="first";
-  document.getElementById("update-Last-name").value="last";
-  document.getElementById("update-Email").value="email";
-  document.getElementById("update-Phone").value="phone";
-}
-//update button in box
-function updateButton(){
-  document.getElementById("update-result").innerHTML="update successful";
-  document.getElementById("update-list").hidden=false;
-  document.getElementById("update-box").hidden=true;
-}
-function updateCancelButton(){
-  document.getElementById("update-list").hidden=false;
-  document.getElementById("update-box").hidden=true;
-}
-function delSearchButton(){
-  let srch = document.getElementById("delete-search-text").value;
-  resetLists();
-  
-  for(let i = 0; i<2; i++){
-    createDeleteList(i);
-  }
-}
-function createDeleteList(id){
-  let list = document.getElementById("delete-list");
-
-  var doc = document.createElement("div");
-  doc.setAttribute("class","tabcontainer");
-  doc.setAttribute("id", "delete-"+id);
-  //check checkboxes
-  if(document.getElementById("delete-first-name").checked)
-    doc.append("fname\n");
-  if(document.getElementById("delete-last-name").checked)
-    doc.append("lname\n");
-  if(document.getElementById("delete-email").checked)
-    doc.append("email\n");
-  if(document.getElementById("delete-phone").checked)
-    doc.append("phone\n");
-  
-  doc.appendChild(createDeleteButton(id));
-  
-  list.appendChild(doc);
-}
-function createDeleteButton(id){
-  var btn = document.createElement("button");
-  var btnId = "delete-button-"+id;
-
-  btn.setAttribute("id", btnId);
-  btn.setAttribute("type", "button");
-  btn.setAttribute("class", "buttons");
-  btn.setAttribute("onClick", "deleteContactButton(this.id)");
-  
-  btn.append("Delete");
-  return btn;
-}
-//delete button in list
-function deleteContactButton(id){
-  deleteid=id.replace("button-","");
-  //show confirm message
-  document.getElementById("confirm-delete-box").style.display='block';
-}
-//cancel button in delete confirm box
-function cancelDeleteButton(){
-  document.getElementById('confirm-delete-box').style.display='none'
-}
-//delete button in delete confirm box
-function confirmDeleteButton(){
-  document.getElementById(deleteid).remove();
-  document.getElementById('confirm-delete-box').style.display='none'
-  document.getElementById("delete-result").innerHTML=deleteid;
-}
 function logoutButton(){
   window.location.href = 'index.html';
 }
+function createContactList(){
+  const tableBody = document.getElementById("contactTable").getElementsByTagName("tbody")[0];
+	
+	for(let i=0; i<5; i++){
+    const row = document.createElement("tr");
+
+    row.innerHTML = `
+      <td>First Name</td>
+      <td>Last Name</td>
+      <td>Phone Number</td>
+	    <td>Email@email.com</td>
+      <td class='table-buttons'>
```

Comment:
what is this?

Reply:
I was using it for creating rows in the table for styling